packages/cloud-cognitive/src/components/Datagrid/Datagrid/DatagridRow.js

```diff
@@ -23,6 +23,7 @@ const DatagridRow = (datagridState) => {
     <TableRow
       className={cx(`${blockClass}__carbon-row`, {
         [`${blockClass}__carbon-row-expanded`]: row.isExpanded,
+        [`bx--data-table--selected`]: row.isSelected,
```

Comment:
This line should import the carbon prefix from our settings like the following:

```jsx
import { carbon } from '../../settings';

...
...
[`${carbon.prefix}--data-table--selected`]: row.isSelected,
...
...
```

Reply:
@matthewgallo `${carbon.prefix}` class updated.